src/lib.rs

```diff
@@ -203,23 +203,60 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
+    FontNotFound(FontDesc),
+
+    /// Tried to get size metrics from a Face that didn't have a size.
+    MissingSizeMetrics,
+
+    MissingGlyph(RasterizedGlyph),
```

Comment:
```suggestion
    /// The glyph could not be found in any font.
    MissingGlyph(RasterizedGlyph),
```

Alternatively:

```
/// Unable to find the glyph in any font.
```

But I personally like to mix things up a bit, instead of just always starting with the same words? Seems a bit redundant otherwise.